### ISSUE TITLE
Add `paddle.nn.functional.activation` NN Activation Functions to PaddlePaddle Frontend

### ISSUE THREAD

**Issue by hmahmood24** [unifyai/ivy#15050]
Add [NN Activation Functions](https://www.paddlepaddle.org.cn/documentation/docs/en/api/index_en.html) to PaddlePaddle Frontend

> Please keep in mind that the proper way to link an issue to this list is to comment "- [ ] #issue_number" while the issue's title only includes the name of the function you've chosen.

- [ ] celu
- [ ] elu
- [ ] elu_
- [ ] gelu
- [ ] hardshrink
- [ ] hardtanh
- [ ] hardsigmoid
- [ ] hardswish
- [ ] leaky_relu
- [ ] log_sigmoid
- [ ] maxout
- [ ] prelu
- [ ] relu
- [ ] relu_
- [ ] relu6
- [ ] selu
- [ ] sigmoid
- [ ] silu
- [ ] softmax
- [ ] softmax_
- [ ] softplus
- [ ] softshrink
- [ ] softsign
- [ ] swish
- [ ] mish
- [ ] #15042
- [ ] tanh_
- [ ] tanhshrink
- [ ] thresholded_relu
- [ ] log_softmax
- [ ] glu
- [ ] gumbel_softmax
- [ ] rrelu

**Comment by sarakarati** [comment]
- [ ] #15565

**Comment by KSANTHOSH200** [comment]
- [ ] #15794

**Comment by maher-mohsen** [comment]
"- [ softmax] #15050

**Comment by KSANTHOSH200** [comment]
- [ ] #15893

**Comment by KSANTHOSH200** [comment]
- [ ] #15894

**Comment by sarakarati** [comment]
Softplus 15896

**Comment by sarakarati** [comment]
"-softplus #15896"

**Comment by sarakarati** [comment]
"-[softplus] #15896"

**Comment by sarakarati** [comment]
softplus #15894

**Comment by KSANTHOSH200** [comment]
- [ ] #15897

**Comment by KSANTHOSH200** [comment]
- [ ] #15899

**Comment by KSANTHOSH200** [comment]
- [ ] #15893

**Comment by KSANTHOSH200** [comment]
- [ ] #15900

**Comment by KSANTHOSH200** [comment]
- [ ] #15901

**Comment by KSANTHOSH200** [comment]
- [ ] #15893

**Comment by maher-mohsen** [comment]
![log_softmax] #15909

**Comment by maher-mohsen** [comment]
#15916

**Comment by sachelsout** [comment]
- [ ] #15945

**Comment by luizgubarichello** [comment]
- [ ] #15949

**Comment by mobley-trent** [comment]
- [ ] #15977

**Comment by Nordiniv** [comment]
- [ ] #15982

**Comment by AlexandraPosta** [comment]
- [ ] #16017

**Comment by sachelsout** [comment]
- [ ] #16180

**Comment by sachelsout** [comment]
- [ ] #16185

**Comment by sachelsout** [comment]
- [ ] #16189

**Comment by sachelsout** [comment]
- [ ] #16197

**Comment by sachelsout** [comment]
- [ ] #16240

**Comment by sachelsout** [comment]
- [ ] #16243

**Comment by sachelsout** [comment]
- [ ] #16245

**Comment by sachelsout** [comment]
- [ ] #16249

**Comment by sachelsout** [comment]
- [ ] #16293

**Comment by sachelsout** [comment]
- [ ] #16337

**Comment by sachelsout** [comment]
- [ ] #16343

**Comment by sachelsout** [comment]
- [ ] #16345

**Comment by Adeola23** [comment]
#16456

**Comment by sasori-s** [comment]
<softmax> #16473

**Comment by shivam-151197** [comment]
tanh_ #16478

**Comment by RohitPawar2406** [comment]
- [ ] #16497

**Comment by menatallh** [comment]
#17662

**Comment by peterwisu** [comment]
sigmoid #18312

**Comment by aadya940** [comment]
#18340

**Comment by aadya940** [comment]
#18342

**Comment by peterwisu** [comment]
elu #18493

**Comment by Lucifer60** [comment]
swish #15900

**Comment by sachelsout** [comment]
- [ ] #18944

**Comment by sachelsout** [comment]
- [ ] #18946

**Comment by huzi6181** [comment]
- [ ] #19092

**Comment by AhmadAli016** [comment]
- [ ]<elu> #19193

**Comment by xXMasterXx07** [comment]
#19295

**Comment by xXMasterXx07** [comment]
#19296

**Comment by xXMasterXx07** [comment]
- [ ] #19296

**Comment by owais8** [comment]
- [ ] #19327

**Comment by smitkalathiya** [comment]
-[gumbel_softmax] #19381

**Comment by pranjanpr** [comment]
#19605

**Comment by shafir360** [comment]
- [ ] #19645

**Comment by shafir360** [comment]
- [ ] #19645

**Comment by Krish2002** [comment]
can i pick up sigmoid #19092 @hmahmood24

**Comment by arshPratap** [comment]
- [ ] #22315

**Comment by imessam** [comment]
- [] #22623

**Comment by imessam** [comment]
@ReneFabricius is the relu task is now assigned to me or there's another step I have to do ?

**Comment by imessam** [comment]
- [] #22623

**Comment by jieunboy0516** [comment]
- [ ] #22623

**Comment by arshPratap** [comment]
- [ ] #22893

**Comment by arshPratap** [comment]
- [ ] #22894

**Comment by DpavanM** [comment]
- [ ] #23008

**Comment by abdul-rehman18** [comment]
#25768

**Comment by OguzTNCR** [comment]
#26293

**Comment by HassanAkhlaq-01** [comment]
Hi. Will be doing elu_ (https://github.com/unifyai/ivy/issues/18944) for my coding challenge.
